core/bones/file.py

```diff
@@ -131,31 +136,33 @@ class FileBone(TreeLeafBone):
     "height", and "derived".
     """
 
-    def __init__(
-        self,
-        *,
-        derive: Union[None, Dict[str, Any]] = None,
-        maxFileSize: Union[None, int] = None,
-        validMimeTypes: Union[None, List[str]] = None,
-        **kwargs
-    ):
-        """
+    def __init__(self, *, derive: Union[None, Dict[str, Any]] = None, maxFileSize: Union[None, int] = None,
+                 validMimeTypes: Union[None, List[str]] = None, **kwargs):
```

Comment:
Can you please keep the previous notation? Splitting the signature over multiple lines makes the code more maintainable and readable.